test/test_prototype_transforms.py

```diff
@@ -66,10 +66,10 @@ def parametrize_from_transforms(*transforms):
 class TestSmoke:
     @parametrize_from_transforms(
         transforms.RandomErasing(p=1.0),
-        transforms.HorizontalFlip(),
         transforms.Resize([16, 16]),
         transforms.CenterCrop([16, 16]),
         transforms.ConvertImageDtype(),
+        transforms.RandomHorizontalFlip(),
```

Comment:
Why this change? We should support the idiom of doing the image transforms prior converting the image to floats.

Reply:
This is not a composed transform, but rather a list of transforms to test.

Reply:
lol, I need another coffee